opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -161,82 +161,49 @@ message MetricDescriptor {
-    //      number of requests received over the interval of time t_0 to
-    //      t_0+1 with a value of 3.
-    //   6. A request is received, the system measures 1 request.
-    //   7. A request is received, the system measures 1 request.
-    //   8. The 1 second collection cycle ends. A metric is exported for the
-    //      number of requests received over the interval of time t_0+1 to
-    //      t_0+2 with a value of 2.
-    DELTA = 2;
-
-    // CUMULATIVE is a metric whose values are the aggregation of
-    // successively made measurements from a fixed start time until the last
-    // reported measurement. This means that current values of a CUMULATIVE
-    // metric depend on all previous measurements since the start time.
-    // Because of this, the sender is required to retain this state in some
-    // form. If this state is lost or invalidated, the CUMULATIVE metric
-    // values MUST be reset and a new fixed start time following the last
-    // reported measurement time sent MUST be used.
+    // GROUPING is the default Structure. It SHOULD be used when the data
+    // does not support any operation(s) or it is unknown.
     //
-    // For example, consider a system measuring the number of requests that
-    // it receives and reports the sum of these requests every second as a
-    // CUMULATIVE metric:
+    // Examples of GROUPING data:
+    // - CPU core number that was the hottest.
+    // - If a system was stopped, crashed, or running (represented by the
+    //   integers '0', '1', or '2' respectively).
+    // - The average power consumption of a server.
+    // - CPU fan speed.
+    GROUPING = 0;
+
+    // ADDING data can be combined meaningfully when added together.
     //
-    //   1. The system starts receiving at time=t_0.
-    //   2. A request is received, the system measures 1 request.
-    //   3. A request is received, the system measures 1 request.
-    //   4. A request is received, the system measures 1 request.
-    //   5. The 1 second collection cycle ends. A metric is exported for the
-    //      number of requests received over the interval of time t_0 to
-    //      t_0+1 with a value of 3.
-    //   6. A request is received, the system measures 1 request.
-    //   7. A request is received, the system measures 1 request.
-    //   8. The 1 second collection cycle ends. A metric is exported for the
-    //      number of requests received over the interval of time t_0 to
-    //      t_0+2 with a value of 5.
-    //   9. The system experiences a fault and loses state.
-    //   10. The system recovers and resumes receiving at time=t_1.
-    //   11. A request is received, the system measures 1 request.
-    //   12. The 1 second collection cycle ends. A metric is exported for the
-    //      number of requests received over the interval of time t_1 to
-    //      t_0+1 with a value of 1.
-    CUMULATIVE = 3;
+    // Examples of ADDING data:
+    // - CPU core temperature.
```

Comment:
Temperature belongs in GROUPING, I think. We would average it, not sum it, I believe.